### PR TITLE
fix(multiprocess): startup multiprocess even when no direct appenders

### LINKED ISSUE #1161: Multiprocess appender does not work, master doesn't start
The appender lazy-loading mechanism introduced here ~2 years ago broke the multiprocess appender.

Commit here: https://github.com/log4js-node/log4js-node/commit/3cb7f52fce3d4721685649d33999f36443477eac

### PR DESCRIPTION
**Pull request by nicojs**
Fixes #1161

### CODE REVIEW

**Comment by nicojs** [comment]
Thanks 😊 @peteriman . Any plans to release a new patch version?

File: lib/appenders/index.js [hunk 1]
```diff
@@ -95,9 +95,9 @@ const setup = (config) => {
     usedAppenders.push(...category.appenders)
   });
   Object.keys(config.appenders).forEach((name) => {
-    // dodgy hard-coding of special case for tcp-server which may not have
+    // dodgy hard-coding of special case for tcp-server and multiprocess which may not have
     // any categories associated with it, but needs to be started up anyway
-    if (usedAppenders.includes(name) || config.appenders[name].type === 'tcp-server') {
+    if (usedAppenders.includes(name) || config.appenders[name].type === 'tcp-server' || config.appenders[name].type === 'multiprocess') {
```
Comment (peteriman) [inline lib/appenders/index.js:100]: Fixed eslint warnings in #1165

**Comment by peteriman** [comment]
> Thanks 😊 @peteriman . Any plans to release a new patch version?

Doing it at the moment. :technologist:

**Comment by peteriman** [comment]
Published `log4js@6.4.1`.

**Comment by nicojs** [comment]
Thanks! Will try it out and if I don't find anything else will close the other issue

**Pull request merged**